getting-started/installing-maestro/windows.md

```diff
@@ -82,8 +82,8 @@ maestro --version
     * Create a new directory in your home directory.
 
     ```
-    ~ $ mkdir Android
-    ~ $ cd Android
+    $ mkdir Android
+    $ cd Android
```

Comment:
Should we add a `cd ~` to make sure they're in the right place?

Reply:
Have added a few more commits that should resolve this commit and improve overall developer experience when setting up Maestro on Windows